src/pages/Home.jsx

```diff
@@ -7,86 +7,84 @@ function Home () {
+              <div className='flex h-10 px-20 items-center text-center justify-center bg-green-700 self-stretch rounded-full transition-all hover:text-green-700 hover:bg-transparent hover:border-solid hover:border-4 hover:border-green-700'>
                   Our Services
               </div>
-              </Link>
-            </div>
+            </Link>
           </div>
         </div>
       </div>
       {/* web version */}
+
       <div className='hidden lg:flex flex-row'>
-      <div className='flex flex-row'>
         <img
-          className='w-1/3'
+          className='flex-2 flex-shrink-1 flex-grow-0 w-1/2'
           src='/images/hero.png'
           alt='Freshly mowed vibrant green grass showcasing immaculate grooming. In the foreground, beautiful flower bushes bloom with a variety of white, red, and pink flowers.'
         />
-        <div className='w-full'>
-          <div className='flex flex-row absolute right-0 top-0 pt-2'>
-            <p className='font-normal font-header text-4xl mr-4'>
+        <div className='flex-2 flex-shrink-1 flex-grow-0 w-1/2 md:px-24'>
+          <div className='flex flex-row justify-center md:py-4'>
+            <p className='font-normal font-header text-4xl'>
               Call us today!
             </p>
             <p className='font-normal font-header text-4xl text-green-700 mr-4'>
               123.456.7890
             </p>
           </div>
-          <div className='flex flex-col mx-auto mt-18 py-36 text-center items-center'>
+          <div className='flex flex-col mx-auto mt-18 py-36 md:my-28 justify-center text-center items-center'>
             <div className='mx-2'>
               <BaseHeading level={1} text='Yorkshire Woods Ground Crew' />
               <p className='font-normal my-6 italic text-4xl text-zinc-500'>
                 Detroit&apos;s Finest Lawn Care
               </p>
             </div>
-            <div className='flex flex-row font-normal text-lg md:text-xl lg:text-2xl mb-14 divide-x-2 divide-zinc-500'>
+            <div className='flex flex-col font-normal text-lg md:text-xl lg:text-2xl mb-14 py-9 gap-2 divide-zinc-500'>
               <h3 className='px-2'>Personalized Service</h3>
               <h3 className='px-2'>Jobs Of All Sizes</h3>
               <h3 className='px-2'>Quality Guaranteed</h3>
             </div>
               <Link
                 to='#services-section'
-                className='text-2xl lg:text-3xl text-white font-bold'
+              className='text-2xl lg:text-3xl text-white font-bold'
               >
-            <div className='flex items-center text-center justify-center bg-green-700 p-2.5 gap-2.5 rounded-lg w-3/5 lg:w-[463px] lg:h-[85px]'>
+              <div className='flex h-16 md:px-20 items-center text-center justify-center bg-green-700 self-stretch rounded-full transition-all hover:text-green-700 hover:bg-transparent hover:border-solid hover:border-4 hover:border-green-700'>
```

Comment:
Issue: A `div` has no semantic meaning, but the text inside looks meaningful. What would be a better tag for this? My guess is a heading tag of some level.